### PR TITLE
Map extension to categories

### PR DESCRIPTION
**Pull request by valfirst**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,12 @@
     "engines": {
         "vscode": "^1.74.0"
     },
-    "categories": [],
+    "categories": [
+        "Debuggers",
```
Comment (uarlouski) [inline package.json:17]: not sure if its appropriate for us

Comment (valfirst) [reply package.json:17]: you can run tests, right?

Comment (uarlouski) [reply package.json:17]: for me it sounds like debugging similar to what we have in IDE :)

**Review by ikalinin1 — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (valfirst) [reply package.json:17]: we don't have "runners" category, thus (I assume) debugging means: run, troubleshoot, fix, run again, etc.

**Pull request merged**